components/motor/gpio/motor_encoder.go

```diff
@@ -175,6 +175,19 @@ type EncodedMotorState struct {
 
 // RPMMonitorStart starts the RPM monitor.
 func (m *EncodedMotor) rpmMonitorStart() {
+	if m.loop != nil {
+		return
+	}
+	if len(m.cfg.ControlLoop.Blocks) != 0 {
+		cLoop, _ := control.NewLoop(m.logger, m.cfg.ControlLoop, m)
+		err := cLoop.Start()
+		if err != nil {
+			m.logger.Error(err)
+			return
+		}
+		m.loop = cLoop
+		return
+	}
```

Comment:
I kind of want to take the starting of the control loop out of the constructor so that the motor doesn't immediately turn on without a motor command, so that's why this `if` block would be necessary, but if we keep it in the constructor this can be removed